dotcom-rendering/src/web/lib/useSDC.tsx

```diff
@@ -0,0 +1,40 @@
+import type { ModuleDataResponse } from '@guardian/support-dotcom-components';
+import {
+	getBanner,
+	getEpic,
+	getLiveblogEpic,
+	getPuzzlesBanner,
+} from '@guardian/support-dotcom-components';
+import type {
+	BannerPayload,
+	EpicPayload,
+} from '@guardian/support-dotcom-components/dist/dotcom/src/types';
+import type { SWRConfiguration } from 'swr';
+import useSWR from 'swr';
+
+const options: SWRConfiguration = {
+	revalidateIfStale: false,
+	revalidateOnFocus: false,
+	revalidateOnReconnect: false,
+};
+
+/**
+ * Hooks for making requests to the support-dotcom-components API
```

Comment:
Currently only `useSDCLiveblogEpic` is used, as the other components use the SDC library directly. But in future we should migrate them to use these hooks so that we're consistent in using SWR